portfolio/src/main/webapp/script.js

```diff
@@ -32,48 +32,56 @@ function displayLocation(name) {
  * Hides label and changes opacity back to 1 when mousing out of image. 
  */
 function hideLocation(name) {
-  var [label, img] = getImage(name); 
+  const [label, img] = getImage(name);
   label.style.display = 'none';
   img.style.opacity = 1; 
 }
 
 /**
- * Async function to fetch server content and add it to DOM.
+ * Add server content to DOM.
  */
-async function getServerContent() {
-  var num = "?num=" + document.getElementById('display-num').value;
-  var order = "&order=" + document.getElementById('display-order').value;
-  var user = "&user=" + document.getElementById('display-user').value;
-
-  const response = await fetch('/data' + num + order + user);
-  const content = await response.text();
-  var obj = JSON.parse(content); 
+async function displayServerContent() {
+  const serverResults = await getServerContent();
```

Comment:
I think you might be able to do const `const {comments, totalComments} = await getServerContent()` so that below you can just use the variables `comments` and `totalComments` to make it more readable.